FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Api/Dicom/DicomApiDataClient.cs

```diff
@@ -0,0 +1,190 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Extensions.Logging;
+using Microsoft.Health.Fhir.Synapse.Common;
+using Microsoft.Health.Fhir.Synapse.Common.Authentication;
+using Microsoft.Health.Fhir.Synapse.Common.Configurations;
+using Microsoft.Health.Fhir.Synapse.Common.Logging;
+using Microsoft.Health.Fhir.Synapse.DataClient.Exceptions;
+using Microsoft.Health.Fhir.Synapse.DataClient.Extensions;
+using Microsoft.Health.Fhir.Synapse.DataClient.Models;
+using Polly.CircuitBreaker;
+
+namespace Microsoft.Health.Fhir.Synapse.DataClient.Api.Dicom
+{
+    public class DicomApiDataClient : IApiDataClient
+    {
+        private readonly IApiDataSource _dicomApiDataSource;
+        private readonly HttpClient _httpClient;
+        private readonly IDiagnosticLogger _diagnosticLogger;
+        private readonly IAccessTokenProvider _accessTokenProvider;
+        private readonly ILogger<DicomApiDataClient> _logger;
+
+        public DicomApiDataClient(
+            IApiDataSource dataSource,
+            HttpClient httpClient,
+            ITokenCredentialProvider tokenCredentialProvider,
+            IDiagnosticLogger diagnosticLogger,
+            ILogger<DicomApiDataClient> logger)
+        {
+            EnsureArg.IsNotNull(dataSource, nameof(dataSource));
+            EnsureArg.IsNotNullOrEmpty(dataSource.ServerUrl, nameof(dataSource.ServerUrl));
+            EnsureArg.IsNotNull(httpClient, nameof(httpClient));
+            EnsureArg.IsNotNull(tokenCredentialProvider, nameof(tokenCredentialProvider));
+            EnsureArg.IsNotNull(diagnosticLogger, nameof(diagnosticLogger));
+            EnsureArg.IsNotNull(logger, nameof(logger));
+
+            _dicomApiDataSource = dataSource;
+            _httpClient = httpClient;
+            _httpClient.DefaultRequestHeaders.Add("Accept", "application/dicom+json");
```

Comment:
Is it possible that httpclient has the `Accept` header already? do we need to check it before `add`?

Reply:
The header accepts multiple values and all of them would take effect. Unit tests are added correspondingly.